plugin/ale.vim

```diff
@@ -299,35 +299,3 @@ nnoremap <silent> <Plug>(ale_go_to_definition) :ALEGoToDefinition<Return>
 nnoremap <silent> <Plug>(ale_go_to_definition_in_tab) :ALEGoToDefinitionInTab<Return>
 nnoremap <silent> <Plug>(ale_find_references) :ALEFindReferences<Return>
 nnoremap <silent> <Plug>(ale_hover) :ALEHover<Return>
-
-" Set up autocmd groups now.
-call ale#autocmd#InitAuGroups()
-
-" Housekeeping
-
-augroup ALECleanupGroup
```

Comment:
Leave the deprecated functions in for now. We can remove them after we get version 1.9 out. I'd leave a bit of a grace period after 1.9 is out. We can probably release 1.9 this month, leave these deprecated things in for another month, and push a commit to remove them in July.